README.md

```diff
@@ -58,6 +58,7 @@ an "image" attachment on a `photos` table this would be an `image_data` column:
 
 ```
 $ rails generate migration add_image_data_to_photos image_data:text # or :jsonb
+# don't forget to add a gin index to image_data if using jsonb and you want search-ability
```

Comment:
I agree with @jrochkind in referring the user to Postgres docs. Part of the issue is the way the above sentence comes across -"don't forget to add...". Re-phrasing it to something like "If you are experiencing slow queries on image_data (json/jsonb type), consider adding a gin index (refer to docs...)" will change it to be a tip than a recommendation or must do item. Generally I'm ok to add this if it doesn't cascade to adding it in other forms and many other places for consistency because it's more of a database performance issue than a Shrine file management issue.

Did you have a use case where you needed gin indexes to speed up queries on image_data?

Reply:
> Did you have a use case where you needed gin indexes to speed up queries on image_data?

Yes, that is correct. I want to search by file name, quickly - I may not be alone in this.

As alluded to, documentation presents a variety of cost-benefits: 

- ease-of-use vs.
- over-documentation / clutter vs.
- maintenance costs

Reply:
Ok let's rephrase it as discussed and we can add it unless others object.